e2e/tests/constants.ts

```diff
@@ -1 +1,6 @@
-export const ALL_PAGES = [{ url: '/', name: 'home' }]
+export const ALL_PAGES = [
```

Comment:
It would be good to get Eleventy generating these.

Reply:
It's possible but a bit complicated because it adds a dependency on the Eleventy build e.g. to linting/type checking (if you import the pages statically). Will do a separate PR for it.